### PR TITLE
Custom command for pr

### PR DESCRIPTION
**Pull request by GarlicB**
- Written in JS. 
- It's located in /bootstrap/command/cc.js.
- To work with the gulp task, edited gulpfile.babel.js (adding task 'cc', 'cp').
- Added the gulp option '--silent' in absolute.sh.

Use the 'readline' module for multiple inputs, But I think this part is quite rough. 
(This is why it took a long time from the last time.)

Always appreciate your review.

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -139,17 +141,16 @@ gulp.task('bootstrap_test', () => {
...
     context: path.resolve(__dirname, 'client'),
-    entry: {
-      bundle: './app.js',
-      sw: './service-worker.js'},
+    entry: './app.js',
     output: {
-      path: path.resolve(__dirname, 'out', 'client'),
-      filename: '[name].js'},
+      path: path.resolve(__dirname, 'out', 'client', 'javascript'),
+      filename: 'bundle.js'},
```
Comment (daehyunjimmy) [inline gulpfile.babel.js]: - please check duplicated app.js entry
- need to keep [name].js filenama and path to generate right service worker file

Comment (GarlicB) [inline gulpfile.babel.js]: thanks, i'll check.

File: bootstrap/absolute.sh [hunk 2]
```diff
@@ -47,4 +47,9 @@ done
 # TODO(nadongguri): this env vir should be set not here but before gulp command
 export BABEL_CACHE_PATH=$(absolute_path)/.babel-cache.json
 
-gulp $@
+# ccl requires several steps. it need to turn off the gulp message
+if [ "$@" = "cc" ]; then
+  gulp $@ --silent
```
Comment (nadongguri) [inline bootstrap/absolute.sh]: As far as I heard, putting silent option is to avoid async call sequence. But is there other ways to handle?

Comment (GarlicB) [inline bootstrap/absolute.sh]: Easy to say, the reason I put the '--silent' option on the 'cc' task is that the gulp message is printed every time when typing an answer. I'll see another way.

**Comment by nadongguri** [comment]
Would you remove .babel-cache.json?

**Comment by nadongguri** [comment]
I don't know why but import {inputRL, rl} from './bootstrap/command/cc'  blocks ending of gulp process.

**Comment by GarlicB** [comment]
Close this, because my request have some problem about running project. 
I will summarize my thoughts and write new issue.

**Pull request closed without merging**